docs/mesos/ManualInstallCalico.md

```diff
@@ -0,0 +1,63 @@
+# Manually Install the Calico Mesos Plugin
+This guide will walk you through manually downloading and installing the Calico-Mesos network plugin for your Mesos deployment. 
+
+The Calico-Mesos plugin must be installed with Calico's core services running on every Agent node in your cluster in order to take advantage of Calico networking. 
+
+*This guide assumes you already have Mesos + Net-Modules installed on your Agent(s).* 
```

Comment:
Turn `Mesos + Net-modules` into a link to the Mesos+netmodules guide section of the toplevel readme